baseband/io/__init__.py

```diff
@@ -61,8 +61,14 @@ def __getattr__(attr):
 
         # Note: again do not presume the entry points exist, since we may
         # be in a pure source checkout.
+
+        if hasattr(entry_points(), 'select'):
+            selected_entry_points = entry_points().select(group='baseband.io')
```

Comment:
Might as well use `entry_points(group='baseband.io')`, which seems nicer (though apparently the two are exactly equivalent)

Reply:
Made changes and swapped to try/except also!